docs/data/forecast.json.jl

```diff
@@ -0,0 +1,35 @@
+using Pkg
+Pkg.add("HTTP")
+Pkg.add("JSON")
+
+# It will be useful to create and commit a Project.toml to manage dependencies.
+# Activate environment as shown below and add packages. Once done, remove
+# any package add statements. Committing the Manifest.toml file is generally
+# not recommended.
+# Pkg.activate(@__DIR__)
+# Pkg.instantiate()
```

Comment:
I'd like to understand this comment. Is this something that needs to run once to set up the HTTP and JSON packages? If so, how would the same code run both locally and in continuous deployment?

I have the same question with R's [install_packages](https://search.r-project.org/CRAN/refmans/easypackages/html/install_packages.html) which don't seem to have an option "install only when it's not already here" ;-)

Reply:
When Julia packages are added directly (as shown in the example using `Pkg.add`) without creating a specific environment, they become available in the global scope. This is generally not a good practice. Hence, adding a `Project.toml` is recommended (see [example](https://github.com/vnegi10/WeatherReport.jl/blob/main/Project.toml)), which contains the dependencies and optional sections to impose compatibility constraints.

The code can be run any number of times, both locally and during CI. If the packages have been downloaded before on a system, they will simply be added to the scope and not downloaded again. For instance, you will see the downloads happening only the first time locally. However, since CI runs in a new environment every time, packages will always be downloaded there.

I added this comment to make the user aware of a practice that I like to follow, which is having a `Project.toml` also along side the data loader. In that case, packages should be loaded as follows: 
```
Pkg.activate(@__DIR__)
Pkg.instantiate()
```

I didn't commit the `Project.toml` since I wasn't sure if you wanted it to be part of the repo.